site/ja/tutorials/quickstart/beginner.ipynb

```diff
@@ -207,19 +203,200 @@
       },
       "outputs": [],
       "source": [
-        "model.fit(x_train, y_train, epochs=5)\n",
+        "predictions = model(x_train[:1]).numpy()\n",
+        "predictionspredictions = model(x_train[:1]).numpy()\n",
+        "predictions"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "`tf.nn.softmax` 関数はこれらのロジットをそれぞれのクラスにである\"確率\"に変換します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "0egSSdWsh5cp"
+      },
+      "outputs": [],
+      "source": [
+        "tf.nn.softmax(predictions).numpy()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "a87oUHS3h5cs"
+      },
+      "source": [
+        "Note: `tf.nn.softmax` をネットワークの最終レイヤーに対する活性化関数として組み込むことは可能です。これはモデルの出力に対して直接的に解釈できるようにする一方で、ソフトマックスの出力を用いる場合、すべてのモデルに対して正確で数値的安定に損失を計算することが不可能なため、このアプローチは推奨されません。"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "dhVMTSPXh5cs"
+      },
+      "source": [
+        "`losses.SparseCategoricalCrossentropy` 損失関数はロジットからなるベクトルと `True` を入力にとり、それぞれの標本についてクラスごとに損失のスカラーを返します。"
```

Comment:
[weak suggestion]
```suggestion
        "`losses.SparseCategoricalCrossentropy` 損失関数は、ロジットと `True` の インデックスに関するベクトルを入力にとり、それぞれの標本についてクラスごとに損失のスカラーを返します。"
```

Based on `scce(y_true, y_pred).numpy()` at https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy

Reply:
Thanks, I'd like to use your suggestion.